packages/gatsby-plugin-mdx/utils/create-fake-file-node.js

```diff
@@ -26,14 +25,8 @@ exports.createFileNode = async (
   const stats = fs.statSync(slashedFile.absolutePath)
   let internal
   if (stats.isDirectory()) {
-    const contentDigest = crypto
-      .createHash(`md5`)
-      .update(
-        JSON.stringify({ stats: stats, absolutePath: slashedFile.absolutePath })
-      )
-      .digest(`hex`)
     internal = {
-      contentDigest,
+      contentDigest : createContentDigest(obj),
```

Comment:
Don't see an `obj` variable here. Shouldn't that be `{ stats: stats, absolutePath: slashedFile.absolutePath }`?

Reply:
Thanks for pointing out, I will fix it.

Reply:
https://github.com/gatsbyjs/gatsby/pull/20239/commits/8335edfeda7200070b6907a503b2e5ce2d02f0e8 fixes this